internal/aws/containerinsight/const.go

```diff
@@ -257,4 +261,18 @@ func init() {
 		ContainerRestartCount: UnitCount,
 		RunningTaskCount:      UnitCount,
 	}
+
+	PodPhaseMetricNames = map[corev1.PodPhase]string{
```

Comment:
I would keep these maps in podstore itself since I dont see these being used anywhere else.

Reply:
done